### PR TITLE
Try to expand the matrix more, reconfiguring to let travis handle the…

### PR DESCRIPTION
**Pull request by liquidpele**
… matrix generation more for us.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +1,44 @@
 language: python
 sudo: True
 cache: pip
-python:
-    - '3.6'
 os:
     - linux
 dist: xenial
+python:
+  - 2.7
```
Comment (auvipy) [inline .travis.yml]: this is django 1.11 only so includ in django 1.11 was enough

Comment (liquidpele) [reply .travis.yml]: what do you mean?

Comment (auvipy) [reply .travis.yml]: Ask for review before merging from now on.

Comment (liquidpele) [reply .travis.yml]: Sure, I can do that, but what do you mean by "this is django 1.11 only so includ in django 1.11 was enough" ?

**Pull request merged**